### PR TITLE
Reduced sourcecode handler

### PR DESCRIPTION
**Pull request by CommanderStorm**
## Proposed Changes (include Screenshots if possible)

- This PR removes the source code handlers
- Reasoning:
  - For debugging, I don't consider this useful
  - Why keep something that is not useful?

## How to test this PR

1. Discuss if sourcecode is not needed instead

## How has this been tested?

- internal monolouge, that this feature is not needed

## Checklist:

- [x] I have updated the documentation / No need to update the documentation
- [x] I have run the linter

### CODE REVIEW

**Comment by octycs** [comment]
Since it's GPL (as said in the doc), we don't need to have it. For me it's kind of equal whether to keep it or to remove it. It follows the idea of GPL to provide access to the source code (so it wasn't intended for debugging anyway), but the repo is also directly accessible from the NavigaTUM website. The only thing is that people without access to the deployment might not know which commit is currently active. But if you say it's rather unnecessary or even standing in the way, I am okay to remove it.

**Comment by CommanderStorm** [comment]
> The only thing is that people without access to the deployment might not know which commit is currently active.

Normally I would respond that this never happens and if it does it would be my job to spot this via the alerting we have.
But #352 has shown me some gaps in my alerting for which I really dont have a good way to adress them, so you are probably right.
Thinking more about this, I think this should be combining the `health` and the `source_code` endpoint into a `status` endpoint.

File: server/src/main.rs [hunk 1]
```diff
@@ -9,20 +9,7 @@ mod utils;
...
-        Err(_) => gh_base,
-    };
-    HttpResponse::Ok()
-        .content_type("text/plain")
-        .body(github_link)
-}
-
-#[get("/api/health")]
+#[get("/api/status")]
 async fn health_handler() -> HttpResponse {
```
Comment (octycs) [inline server/src/main.rs:13]: Shouldn't this also be adapted?

**Review by octycs — CHANGES_REQUESTED** [review]

**Comment by octycs** [comment]
> Thinking more about this, I think this should be combining the health and the source_code endpoint into a status endpoint.
That's a good compromise :+1:

File: server/src/main.rs (hunk 1 above)
Comment (CommanderStorm) [reply server/src/main.rs]: Ups

**Review by octycs — APPROVED** [review]

**Pull request merged**